test/karma/ReactPlayer.js

```diff
@@ -104,6 +105,14 @@ describe('ReactPlayer', () => {
     it('fires onDuration with delayed load', done => testDurationDelayed(TEST_VIMEO_URL, done))
   })
 
+  describe('Wistia', () => {
+    it('fires onStart', done => testStart(TEST_WISTIA_URL, done))
+    it('fires onPlay', done => testPlay(TEST_WISTIA_URL, done))
+    it('fires onDuration', done => testDuration(TEST_WISTIA_URL, done))
+    it('fires onDuration with delayed load', done => testDurationDelayed(TEST_WISTIA_URL, done))
+    it.skip('fires onPause', done => testPause(TEST_WISTIA_URL, done))
```

Comment:
The `//fast.wistia.com/assets/external/E-v1.js` Wistia JS player is outputting some errors in Firefox. The player works as expected, but it makes the tests have some output. I'll reach out to the Wistia folks to see if they know what the issue may be.